codemon/CodemonReg.py

```diff
@@ -17,22 +17,47 @@ def codemonReg():
         if os.name == 'nt':
             FILE_ATTRIBUTE_HIDDEN = 0x02
             ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
+        writeMeta(path)
+    else:
+        if os.path.exists(os.path.join(path, "meta.json")):
+            print("Meta file already exists.")
+            overwrite = input("Do you want to overwrite it (y/n)? ")
+            if overwrite == 'y':
+                writeMeta(path)
+        else:
+            writeMeta(path)
 
+def writeMeta(path):
     print("Codemon User Registration")
     print("\nPlease enter the following data.\nYour sensitive information is collected for making your life easier while using Codemon.\nIt will be stored on your local machine.\nIt won't be stored or used by Codemon developers for any purpose. ")
     name = input("\nEnter Name : ")
-    codeforces_username = input("Codeforces username (without @) : ")
-    codeforces_password = stdiomask.getpass(prompt="Codeforces password : ")
-    github_username = input("Github username (without @) : ")
-    github_password = stdiomask.getpass(prompt="GitHub Password : ")
+    while (name == ''):
+        print("No Input Detected")
+        name = input("Enter Name : ")
+    codeforces_username = input("\nCodeforces username (without @) : ")
```

Comment:
Do you think if we remove "`(without @)`" here, and write it as `Codeforces username: @ <here user will input>` will look better and more professional ?

Reply:
@ankingcodes I guess, it will be more confusing.
"(without @)" tells them to specifically enter the username without "@"